crates/subspace-networking/examples/get-pieces.rs

```diff
@@ -86,12 +86,26 @@ async fn main() {
 
     let hashed_peer_id = PieceIndexHash::from(crypto::sha256_hash(&node_1.id().to_bytes()));
 
-    let stream_future = node_2.get_pieces_by_range(hashed_peer_id, hashed_peer_id);
-    let mut stream = stream_future.await.unwrap();
-    while let Some(value) = stream.next().await {
-        println!("Piece found: {:?}", value);
+    // obtain closest peers to the middle of the range
+    let key = libp2p::multihash::MultihashDigest::digest(
+        &Code::Identity,
+        &U256::from(hashed_peer_id).to_be_bytes(),
+    );
+    let peer_id = node_2.get_closest_peers(key).await.unwrap()[0];
+
+    let pieces = node_2
```

Comment:
I would create an example of the `get_closest_peers` here and I would also rename it.